data/json/professions.json

```diff
@@ -5762,5 +5768,31 @@
       "male": [ "briefs" ],
       "female": [ "boy_shorts", "sports_bra" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "officeworker",
+    "name": "Office Worker",
+    "description": "You felt years of your life being sapped away, slaving at your computer as you subsisted on a daily pot of coffee, microwave burritos, and an endless supply of crumpled paper to toss.  Perhaps the Cataclysm is a refreshing change of pace…",
```

Comment:
Double space before `Perhaps` and after `toss.`, should remove the comma after `microwave burritos` because it trails with an `and`. Preferential changes, feel free to ignore.

Reply:
That's a fair point, ill change it